http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpWsmDao.scala

```diff
@@ -153,12 +148,19 @@ class HttpWsmDao[F[_]](httpClient: Client[F], config: HttpWsmDaoConfig)(implicit
       groupedLzResources = lzResourcesByPurpose.foldMap(a =>
         a.deployedResources.groupBy(b => (a.purpose, b.resourceType.toLowerCase))
       )
-
-      aksClusterName <- getLandingZoneResourceName(groupedLzResources,
-                                                   "Microsoft.ContainerService/managedClusters",
-                                                   SHARED_RESOURCE,
-                                                   false
+      aksResource <- getLandingZoneResource(groupedLzResources,
+                                            "Microsoft.ContainerService/managedClusters",
+                                            SHARED_RESOURCE
       )
+      aksCluster <- getLandingZoneResourceName(aksResource, useParent = false).map { aksName =>
+        val tagValue = getLandingZoneResourceTagValue(aksResource, "aks-cost-vpa-enabled")
+        val tags: Map[String, Boolean] =
+          Map("aks-cost-vpa-enabled" -> java.lang.Boolean.parseBoolean(tagValue.getOrElse("false")))
+        logger.info(
```

Comment:
Scala comment -- this log statement won't execute because it's wrapped in an `IO` -- it needs to be chained to other `IO`s. I'd probably do something like this if we want to log the AKS cluster and its tags:
```
aksCluster <- getLandingZoneResourceName(aksResource, useParent = false).map { aksName =>
        val tagValue = getLandingZoneResourceTagValue(aksResource, "aks-cost-vpa-enabled")
        val tags: Map[String, Boolean] =
          Map("aks-cost-vpa-enabled" -> java.lang.Boolean.parseBoolean(tagValue.getOrElse("false")))
        AKSCluster(aksName, tags)
      }
      _ <- logger.info(s"Retrieved Landing Zone AKS cluster: ${aksCluster}")
```

Reply:
fixed, thank you! I do think its worth logging the tags for easier debugging. :)